aria_plugin/executor.py

```diff
@@ -12,24 +12,65 @@
 #    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    * See the License for the specific language governing permissions and
 #    * limitations under the License.
-
+import os
+import pickle
+import subprocess
+import sys
+import tempfile
 from threading import Thread
 
 
-from cloudify import ctx
+# This is a hacky way of patching the ruamel before trying to load aria stuff.
+import aria_plugin                                              # noqa: F401
+
 from aria.orchestrator import workflow_runner
+from aria.orchestrator.workflows.executor import process
 from aria.cli.logger import ModelLogIterator
 
-from .environment import Environment
 
+# This is a very hacky way of executing some code before loading any aria
```

Comment:
Instead of sporadic comments, add one big comment explain whole this mechanism, including all the flow. Be specific.

Reply:
And add this to the commit message. So it should be good =)